languages/ruby/exercises/concept/conditionals-ternary/.docs/instructions.md

```diff
@@ -0,0 +1,39 @@
+In this exercise you will rewrite `if/else` statements from a movie theater's website into ternary conditionals.
```

Comment:
This is a great premise! You could elaborate on it more if you wanted.

Also, rather than asking students to rewrite the expression, just ask them to write the expression?

Reply:
I like the premise of working on a movie theatre website. The "rewrite `if/else` statements" bit seems like it may be more suited for the actual tasks (see [this suggestion](https://github.com/exercism/v3/pull/2310/files#r502860694)). Maybe something like this:

```suggestion
In this exercise you've been hired to build your local movie theater's website. As you've just graduated, you're determined to write the cleanest, most concise code you can.
```